src/ucs/vfs/fuse/Makefile.am

```diff
@@ -14,6 +14,11 @@ libucs_fuse_la_LIBADD   = $(FUSE3_LIBS) \
                           $(top_builddir)/src/ucs/libucs.la
 libucs_fuse_la_LDFLAGS  = $(FUSE3_LDFLAGS) -version-info $(SOVERSION)
 libucs_fuse_la_SOURCES  = vfs_fuse.c
+
+EXTRA_DIST = ucx-fuse.pc.in
+pkgconfigdir = $(libdir)/pkgconfig
+pkgconfig_DATA = ucx-fuse.pc
```

Comment:
as discussed offline - pls try to move these inside `$(top_srcdir)/config/module.am`

Reply:
there are issues with conditional processing of makefiles, added separate include file to identify pkg-config variables